source/NVDAObjects/UIA/winConsoleUIA.py

```diff
@@ -419,14 +420,11 @@ def findExtraOverlayClasses(obj, clsList):
 		clsList.append(consoleUIAWindow)
 
 
-class WinTerminalUIA(EnhancedTermTypedCharSupport):
-	def event_UIA_notification(self, **kwargs):
-		"""
-		In an upcoming terminal release, UIA notification events will be sent
-		to announce new text. Block these for now to avoid double-reporting of
-		text changes.
-		@note: In the longer term, NVDA should leverage these events in place
-		of the current LiveText strategy, as performance will likely be
-		significantly improved and #11002 can be completely mitigated.
-		"""
-		log.debugWarning(f"Notification event blocked to avoid double-report: {kwargs}")
+class WinTerminalUIA(UIA):
+	#: Override the role, which is controlTypes.Role.STATICTEXT by default.
+	role = controlTypes.Role.TERMINAL
+	#: New line text is announced using UIA notification events
+	announceNewLineText=False
+
+	def event_UIA_notification(self, notificationKind, notificationProcessing, displayString, activityId):
```

Comment:
Why this override?

Reply:
Because the original implementation uses ui.message. I should copy the foreground check though.

Reply:
What about Braille?

Reply:
LiveText/Terminal doesn't braille, therefore using ui.message here would mean a behavioral difference for braille users.